integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisRestCatalogAdlsIntegrationTestBase.java

```diff
@@ -23,10 +23,10 @@
 import org.apache.polaris.core.admin.model.StorageConfigInfo;
 
 /** Runs PolarisRestCatalogIntegrationBase test on Azure. */
-public abstract class PolarisRestCatalogAzureIntegrationTestBase
+public abstract class PolarisRestCatalogAdlsIntegrationTestBase
     extends PolarisRestCatalogIntegrationBase {
-  public static final String TENANT_ID = System.getenv("INTEGRATION_TEST_AZURE_TENANT_ID");
-  public static final String BASE_LOCATION = System.getenv("INTEGRATION_TEST_AZURE_PATH");
+  public static final String TENANT_ID = System.getenv("INTEGRATION_TEST_ADLS_TENANT_ID");
```

Comment:
@andrew4699 : is that change ok from your POV?

Reply:
Thanks for looking into it @andrew4699! Makes sense, reverted the env rename part.